### ISSUE TITLE
Get a detailed quote does not include "tag" field.

### ISSUE THREAD

**Issue by khanhuynh** [lukePeavey/quotable#27]
## Description

The method `/quotes/:id` does not return the expected response. 

### Steps to reproduce

1. Make a request to
```https
https://api.quotable.io/quotes/HxaMW05ghs7P
```

### Expected result

The response should be an object with the following fields

```ts
content: string
author: string
length: number
tags: string[]
```

### Actual result

Response does not include fields `tags` or `length`

### Environment

NA

**Comment by lukePeavey** [comment]
Thanks. This was actually fixed in a recent commit(64b4c4f859a06ef1bcb77cf9973a633c9c300a33) but the changes had not been deployed yet. (It should be working now)

There is similar issue with the `/authors/:id` endpoint that still needs to be fixed (#28).

**Comment by lukePeavey** [comment]
Could you confirm if this is working on your end now? If so, I’ll go ahead and close this
